tools/asciidoctor/postprocessor.rb

```diff
@@ -0,0 +1,118 @@
+# Copyright (C) 2020  Matthew Glazar
+# See end of file for extended copyright information.
+
+class Copyright < Asciidoctor::Extensions::Postprocessor
+  def process document, output
+    if document.basebackend? 'html'
+      top = %{<!-- Copyright (C) 2020  Matthew Glazar -->
+<!-- See end of file for extended copyright information. -->
```

Comment:
We should also mention that the .html file is generated (unless asciidoctor does that for us?). See website/benchmarks/index.html for example.

Reply:
Asciidoctor adds a meta tag that says generator. I feel like this is pretty lame... I will probably just copy website/benchmarks/index.html's convetions. I may as well do that since we will need it for the man page as well.
```
<head>
<meta charset="UTF-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1.0">
<meta name="generator" content="Asciidoctor 2.0.14">
<title>quick-lint-js: CLI documentation</title>
<link rel="stylesheet" href="./main.css">
</head>
```